csharp/Platform.Data.Tests/LinksConstantsTests.cs

```diff
@@ -41,15 +42,15 @@ public static void ExternalReferencesTest()
             TestExternalReferences<ushort, short>();
             TestExternalReferences<byte, sbyte>();
         }
-        private static void TestExternalReferences<TUnsigned, TSigned>()
+        private static void TestExternalReferences<TUnsigned, TSigned>() where TUnsigned : IUnsignedNumber<TUnsigned>
         {
-            var unsingedOne = Arithmetic.Increment(default(TUnsigned));
+            var unsingedOne = TUnsigned.One;
             var converter = UncheckedConverter<TSigned, TUnsigned>.Default;
             var half = converter.Convert(NumericType<TSigned>.MaxValue);
```

Comment:
Is there a way to make generic conversions in .NET 7, converters may be slower than native conversion.

Reply:
What about 
```
NumericType<TUnsigned>.MaxValue / 2
```
?

Reply:
@FreePhoenix888 division is slow, we already have required value in NumericType<TSigned>.MaxValue